packages/router/router.coffee

```diff
@@ -39,3 +39,22 @@ FlowRouter.route '/organizations/:_id',
     FlowLayout.render 'layout',
       main: 'organizationDetail'
       params: {"organizationId": params._id}
+
+FlowRouter.route '/myDatasets',
```

Comment:
I thought we might want to use the "/datasets" route for a more general view of all the public datasets on Mantle. I called this route "/myDatasets" because it only shows datasets the user owns or has shared with them.